docs/source/technical_tutorials/working_with_r/gedi_access_r.ipynb

```diff
@@ -0,0 +1,906 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    gedi_file &lt;- s3$download_file(Bucket = bucket, Key = key, Filename = file)
</pre><p>instead:</p>

<pre class="ql-syntax" spellcheck="false">s3$download_file(Bucket = bucket, Key = key, Filename = download_file)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/475/discussion/#comment-2024995299'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
@chuckwondo what's returned and why wouldn't you want to store that?

Reply:
stylistically though, catching all responses is best practices in R - particularly if it tells you if the download succeeded or not (does it throw and error on failure?)

Reply:
yes, an error will be thrown, so there's no need to look at the response -- this is perhaps an unusual case, but generally can be treated as though it simply returns `NULL`, thus no need to assign the response to a var